test/localhost.jl

```diff
@@ -0,0 +1,17 @@
+@testset "Localhost" begin
+    wmic_command = "wmic path win32_computersystemproduct get uuid"
+
+    windows_patch = @patch function Sys.iswindows()
+        return true
+    end
+
+    wmic_patch = @patch function read(command::Cmd, String)
```

Comment:
```suggestion
    wmic_patch = @patch function read(command::Cmd, ::Type{String})
```
What you wrote is taking any type as the second argument and assigning it to the variable `String`